src/Command/TestCommand.php

```
@@ -8,22 +8,33 @@
   protected static $defaultName = 'gpstools:test';
   private $factory;
   private $fit_parser;
-
-  public function __construct(GPSTrackFactory $factory, FITParser $fit_parser, FITCSVWriter $fitcsv_writer)
+  private $logger;
+
+  public function __construct(
+    GPSTrackFactory $factory,
+    FITCSVParser $fit_parser,
+    FITCSVWriter $fitcsv_writer,
+    FitCSVTool $fitcsv_tool,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$fitcsv_tool'. (Unused Code Rules)

(UnusedFormalParameter)